mephisto/client/review/review_server.py

```diff
@@ -99,10 +160,28 @@ def data():
             {"finished": finished, "data": current_data if not finished else None}
         )
 
+    @app.route("/all_data_for_current_task")
```

Comment:
I would just call this endpoint `/all_data`. The reason the other endpoint is called `/data_for_current_tasks` is bc it only works for a single task. having both all and current task in the endpoint name is sort of an oxymoron/contradictory.